app/Console/JsCommand.php

```diff
@@ -67,6 +67,7 @@ class JsCommand extends BaseCommand
         'assets/vendor/c3/c3.min.js',
         'assets/vendor/isMobile/isMobile.min.js',
         'assets/vendor/marked/marked.min.js',
+        'assets/vendor/text-caret/index.js',
```

Comment:
Hum.. it's weird, it's included above in `$appFiles`. I will check when I will have 5 minutes.

Reply:
Oops! Missed that. I only checked the vendor because it was inside vendors folder. It’s not working because it’s written “textarea-caret” instead of text-caret. That’s why it is not being included.

Reply:
I've just updated the PR, fixing the reference in `$appFiles`

https://github.com/kanboard/kanboard/pull/4258/files#diff-de951cd95adad4d2ed69fcfdb50460a2L30-L32